contracts/Registry.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity ^0.5.0;
+
+import "./RegistryAbstract.sol";
+
+
+contract Registry is RegistryAbstract {
+    mapping(address => ContributorInfo) contributors;
```

Comment:
As @DaLN mentioned, we can consider using `EnumerableSet` instead of a mapping here in a later PR.

Reply:
will do it in another PR